<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createCollection</class>
 <widget class="QDialog" name="createCollection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>251</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New Collection</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
    background-color: rgb(211,211,211);
    color: rgb(88, 88, 90);
    selection-background-color: rgb(21, 165, 137);
    selection-color: rgb(245, 244, 244);
    font: 16pt
}

QLabel#error_label
{
    color: rgb(220, 130, 30);
}

QLineEdit, QTextEdit, QTableWidget
{
   background-color:  rgb(245, 244, 244)
}

QPushButton
{
	background-color: rgb(21, 165, 137);
    color: rgb(245, 244, 244);
}

QPushButton#home_button, QPushButton#parent_button, QPushButton#refresh_button
{
    background-color: rgb(245, 244, 244);
}

QTabWidget#info_tabs
{
     background-color: background-color: rgb(211,211,211);
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Parent path:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTextBrowser" name="label"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>New name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="coll_path_input"/>
   </item>
   <item>
    <widget class="QLabel" name="error_label">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>createCollection</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>createCollection</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
